pkg/repository/task_redis.go

```diff
@@ -72,6 +72,16 @@ func (r *TaskRedisRepository) IsClaimed(ctx context.Context, workspaceName, stub
 	return exists > 0, nil
 }
 
+const taskResultExpiration = 3600 * 24 // 1 day
+
+func (r *TaskRedisRepository) SetTaskResult(ctx context.Context, workspaceName, stubId, taskId, result string) error {
```

Comment:
We should probably limit the result size here. I can also limit that on the GRPC size as well though.